simulation/environment/WeatherForecasts.py

```diff
@@ -33,16 +33,18 @@ class WeatherForecasts:
             (in seconds), dt + timezone_offset (local time), wind_speed, wind_direction, cloud_cover, description_id)
     """
 
-    def __init__(self, api_key, coords, duration, race_type, weather_data_frequency="daily", force_update=False):
+    def __init__(self, api_key, coords, duration, race_type, golang, weather_data_frequency="daily", force_update=False, origin_coord=None):
```

Comment:
Is there a reason golang is a required variable and doesn't just have a default value of true?